### ISSUE TITLE
Improve: Install SABnzbd on a NAS

### ISSUE THREAD

**Issue by igiddyuup** [sabnzbd/sabnzbd.github.io#55]
## URL: /wiki/installation/install-nas.html

Improvement:

Include Netgear ReadyNas

**Comment by Safihre** [comment]
Do you have a link for that package?

**Comment by igiddyuup** [comment]
I don't but I was using this to set it up..

https://community.netgear.com/t5/Community-Add-ons/Install-NZBGet-17-1-on-ReadyNAS-OS-6-6-0-working/m-p/1163293

I'm having difficulties though.  I'm pretty computer literate, but I don't have much experience with ReadyNas or linux. So I got stuck on a few spots.  I did find a package on readynas' site, but I was unable to figure out how to change the path for the nzb black hole.   after installing following those directions I am still unable to change the nzb dir.  Let me re-phrase I'm able to change it, I just don't understand what I need to change it to, because I'm not familiar with the hierarchy on Readynas.  I've even tried hard coding my internal IP address for the readynas with the folder and it didn't work either.
